### DISCUSSION TITLE
Provide a "bootstrap.efi" File (Like OpenCore Does)

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#54] [General]
**Is your feature request related to a problem? Please describe.**

Just like OpenCore, RefindPlus to be set the default boot manager, could you give a bootstrap.efi to protect RefindPlus to be forever always default from alteration of other boot even manager such as OC?😅😅😅

**Comment by dakanji** [comment]
There is no intention to do this at this time as there are a lot of things involved.
If you are interested in this, please raise to the rEFInd developer (RefindPlus generally maintains upstream compatibility)
